src/init.cpp

```diff
@@ -475,6 +475,7 @@ std::string HelpMessage(HelpMessageMode mode)
         strUsage += HelpMessageOpt("-incrementalrelayfee=<amt>", strprintf("Fee rate (in %s/kB) used to define cost of relay, used for mempool limiting and BIP 125 replacement. (default: %s)", CURRENCY_UNIT, FormatMoney(DEFAULT_INCREMENTAL_RELAY_FEE)));
         strUsage += HelpMessageOpt("-dustrelayfee=<amt>", strprintf("Fee rate (in %s/kB) used to defined dust, the value of an output such that it will cost about 1/3 of its value in fees at this fee rate to spend it. (default: %s)", CURRENCY_UNIT, FormatMoney(DUST_RELAY_TX_FEE)));
     }
+    strUsage += HelpMessageOpt("-dustlimit=<amt>", strprintf(_("Amount of DOGE under which a transaction output is considered dust, in Koinu (default: %u)"), DEFAULT_DUST_LIMIT));
```

Comment:
So we decided we want to express this in fractional dogecoin (as per one of @patricklodder 's recent commits) but this is worded in Koinu, and will report the default (%u formatted DEFAULT_DUST_LIMIT) in koinu (100000000), whereas we have the tests taking arguments like "-dustlimit=0.01" and part of this PR says we're going with expressing these in COIN vs Koinu.  So I think we need to be consistent here?

```suggestion
    strUsage += HelpMessageOpt("-dustlimit=<amt>", strprintf(_("Amount of DOGE under which a transaction output is considered dust, in whole or decimal portion of a COIN. (default: %s)"), FormatMoney(DEFAULT_DUST_LIMIT)));
```

Reply:
Thanks for catching this! I forgot about that message while processing earlier comments! Will revise.